Client/multiplayer_sa/CMultiplayerSA_Vehicles.cpp

```diff
@@ -118,4 +224,9 @@ void CMultiplayerSA::InitHooks_Vehicles()
 {
     EZHookInstall(CDamageManager__ProgressDoorDamage);
     EZHookInstall(CAEVehicleAudioEntity__Initialise);
+
+    // Fix vehicle dirt level
+    EZHookInstall(CVehicleModelInfo_SetClump);
+    EZHookInstall(CVehicleModelInfo_SetDirtTextures);
+    MemCpy((void*)0x85C5E4, &CVehicleModelInfo_Shutdown, 4);
```

Comment:
MemPut

Reply:
Is there any difference? From what I can see, MemPut still calls MemCpy anyway

Reply:
It's the same, but without a size argument